src/common/src/util/prost.rs

```diff
@@ -29,3 +31,44 @@ impl TypeUrl for data::Column {
         "type.googleapis.com/data.Column"
     }
 }
+
+#[inline(always)]
+pub fn is_stream_source(source_node: &SourceNode) -> bool {
+    matches!(source_node.info.as_ref(), Some(SourceInfo::StreamSource(_)))
+}
+
+use risingwave_pb::catalog::{ColumnIndex, TableSourceInfo};
+use risingwave_pb::plan_common::ColumnCatalog;
+use risingwave_pb::stream_plan::source_node::Info as ProstSourceInfo;
+
+use crate::catalog::{ColumnDesc, ColumnId, Schema};
+
+pub fn create_table_info(
```

Comment:
Looks like it's only used by tests, how about putting this function in a new module named `test_utils` in `source::table`?

Reply:
Good idea! I  will do it.